autoload/slime.vim

```diff
@@ -25,7 +25,8 @@ function! s:ScreenSend(config, text)
         \ " -X eval \"readreg p " . g:slime_paste_file . "\"")
   call system("screen -S " . shellescape(a:config["sessionname"]) . " -p " . shellescape(a:config["windowname"]) .
         \ " -X paste p")
-  call system('screen -X colon "
"')
+  call system('screen -X colon "
+"')
```

Comment:
let's revert this line

this [happened](https://github.com/jpalardy/vim-slime/pull/333/files#diff-833e528646c9c9fd0ad2c302578b742106b4b7c5b87cc05c3df24babbb7db5ebR28) last time too 😄